{"uid":"7d6f184ce1985b47","name":"should connect to gateway and then fail after reconnection attempts after gateway is offline","fullName":"test/ContinuumUnavailable.test.ts#Continuum Unavailable Tests should connect to gateway and then fail after reconnection attempts after gateway is offline","historyId":"c35f390068160d5867f6dc0955c8728f:d41d8cd98f00b204e9800998ecf8427e","time":{"start":1756789029732,"stop":1756789053823,"duration":24091},"status":"passed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"framework","value":"vitest"},{"name":"language","value":"javascript"},{"name":"parentSuite","value":"Continuum Unavailable Tests"},{"name":"host","value":"pkrvmccyg1gnepe"},{"name":"thread","value":"vitest-worker-3"},{"name":"package","value":"test.ContinuumUnavailable.test.ts"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":[]},"source":"7d6f184ce1985b47.json","parameterValues":[]}